packages/spacecat-shared-data-access/src/v2/models/base/schema.builder.js

```diff
@@ -378,21 +292,33 @@ class SchemaBuilder {
    */
   #buildIndexes() {
     // eslint-disable-next-line camelcase
-    const { belongs_to, other } = this.rawIndexes;
-
-    // belongs_to indexes come before other indexes
-    const indexes = {
-      ...sortIndexes(belongs_to),
-      ...sortIndexes(other),
-    };
-
-    numberGSIsIndexes(indexes, this.rawIndexes.all);
-
-    this.indexes = {
-      primary: this.rawIndexes.primary,
-      ...(this.rawIndexes.all && { all: this.rawIndexes.all }),
-      ...indexes,
-    };
+    const { all, belongs_to, other } = this.rawIndexes;
+
+    // set the order of indexes
+    const orderedIndexes = [
+      ...all,
+      // eslint-disable-next-line camelcase
+      ...belongs_to,
+      ...other,
+    ];
+
+    this.indexes = { primary: this.rawIndexes.primary };
+
+    let indexCounter = 0;
+    Object.values(orderedIndexes).forEach((index) => { /* eslint-disable no-param-reassign */
```

Comment:
`/* eslint-disable no-param-reassign */` not needed, `index` is never reassigned